extensions/assetLibrary/src/client/Library.tsx

```diff
@@ -0,0 +1,466 @@
+  descriptionLink: formatMessage('Learn more'),
+  installButton: formatMessage('Install Package'),
+  importDialogTitle: formatMessage('Install a Package'),
+  installProgress: formatMessage('Installing package...'),
+  recentlyUsedCategory: formatMessage('Recently Used'),
+  installedCategory: formatMessage('Installed'),
+  updateConfirmationPrompt: formatMessage('Any changes you made to this package will be lost! Are you sure you want to continue?'),
+  updateConfirmationTitle: formatMessage('Update Package'),
+  conflictConfirmationTitle: formatMessage('Conflicting changes detected'),
+  conflictConfirmationPrompt: formatMessage(
+    'This operation will overwrite changes made to previously imported files. Do you want to proceed?'
+  ),
+  removeConfirmationTitle: formatMessage('Remove Package'),
+  removeConfirmationPrompt: formatMessage(
+      'Any changes you made to this package will be lost! In addition, this may leave your bot in a broken state. Are you sure you want to continue?'
+  ),
+  requireEject: formatMessage('To install components, this project must have an ejected runtime. Please navigate to the runtime settings page.'),
+  ejectRuntime: formatMessage('Eject Runtime'),
+  noComponentsInstalled: formatMessage('No packages installed'),
+  noComponentsFound: formatMessage('No packages found. Check extension configuration.'),
+  browseHeader: formatMessage('Browse'),
+  installHeader: formatMessage('Installed'),
+  libraryError: formatMessage('Library Error'),
+  importError: formatMessage('Install Error'),
+}
+
+const Library: React.FC = () => {
+  const [items, setItems] = useState<LibraryRef[]>([]);
+  const [groups, setGroups] = useState<any[]>([]);
+  const { settings, projectId, reloadProject } = useProjectApi();
+  const { setApplicationLevelError, navigateTo, confirm } = useApplicationApi();
+
+  const [ejectedRuntime, setEjectedRuntime] = useState<boolean>(false);
+  const [availableLibraries, updateAvailableLibraries] = useState<any[]>([]);
+  const [installedComponents, updateInstalledComponents] = useState<any[]>([]);
+  const [recentlyUsed, setRecentlyUsed] = useState<any[]>([]);
+  const [programmingLanguageSelection, setProgrammingLanguageSelection] = useState<string>('c#');
+  const [runtimeLanguage, setRuntimeLanguage] = useState<string>('c#');
+
+  const [selectedItem, setSelectedItem] = useState<LibraryRef>();
+  const [working, setWorking] = useState(false);
+  const [addDialogHidden, setAddDialogHidden] = useState(true);
+  const httpClient = useHttpClient();
+  const API_ROOT = '';
+
+  const programmingLanguages = [
+    {
+      key: 'c#',
+      text: 'C#',
+    },
+    {
+      key: 'js',
+      text: 'Javascript',
+    }
+  ];
+
+  const onChangeLanguage = (ev, op, idx) => {
+    setProgrammingLanguageSelection(op.key);
+    return true;
+  }
+
+  const installComponentAPI = async (projectId: string, packageName: string, version: string, isUpdating: boolean) => {
```

Comment:
`async` should be removed since `await` is not used.

Reply:
👍